app/src/main/java/org/schabi/newpipe/util/StreamDialogEntry.java

```diff
@@ -55,6 +55,21 @@
                     new SinglePlayQueue(item), false);
         }
     }),
+    //TODO reduce duplication
+    next(R.string.play_next_stream, (fragment, item) -> {
+        final MainPlayer.PlayerType type = PlayerHolder.getInstance().getType();
+
+        if (type == AUDIO) {
+            NavigationHelper.nextOnVideoPlayer(fragment.getContext(),
+                    new SinglePlayQueue(item), false);
+        } else if (type == POPUP) {
+            NavigationHelper.nextOnPopupPlayer(fragment.getContext(),
+                    new SinglePlayQueue(item), false);
+        } else /* type == VIDEO */ {
+            NavigationHelper.nextOnVideoPlayer(fragment.getContext(),
+                    new SinglePlayQueue(item), false);
+        }
```

Comment:
So this would become:
```suggestion
        NavigationHelper.nextOnVideoPlayer(fragment.getContext(),
                new SinglePlayQueue(item), false);
```